culture/review_culture.md

```diff
@@ -28,27 +28,25 @@ The review process orients around _changes_, which are the units of work being
 reviewed.
 
 A change can take many forms, for example:
-- Adding a new guide or piece of documentation
-- Adding a new module to the codebase
-- Modifying an existing guide or module
-
+- Adding or changing official SourceCred documentation (in [sourcecred/docs](https://github.com/sourcecred/docs))
+- Adding or changing the SourceCred codebase
```

Comment:
nit: ”Adding **to** or changing the SourceCred codebase”?

Reply:
Done, thank you.